spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapter.java

```diff
@@ -77,7 +80,8 @@ private void receiveMessage(PubsubMessage pubsubMessage, AckReplyConsumer consum
 
 		try {
 			sendMessage(MessageBuilder.withPayload(
-					this.pubSubTemplate.getMessageConverter().fromPubSubMessage(pubsubMessage, this.payloadType))
+					// TODO: need to get the message converter from somewhere.
```

Comment:
@artembilan Since the subscriber template now doesn't have the message converter, we should either add it there, even though it will not be used, or have a public method that can do the conversion there, or inject the converter into this class. Any preference?

Reply:
OK. Since there is no conversion anymore, then we should move it here.
Otherwise it does not make sense until we really don't implement the conversion there.